experimental/ExteriorAlgebra/ExteriorAlgebra_naive.jl

```diff
@@ -0,0 +1,95 @@
+export exterior_algebra_naive;  # constructor
+export ContainsDuplicates;   # export just for developing/testing
+
+# Auxiliary function -- worth keeping?
+
+function ContainsDuplicates(L)  # L must be indexable (1:n), and its elems must be sortable
+  n = length(L);
+  if (n < 2)  return false;  end;
+  if (n == 2)  return (L[1] == L[2]);  end;
+  sorted = sort(L);
+  for i=1:n-1  if (sorted[i] == sorted[i+1])  return true;  end;  end;
+  return false;
+end;
+
+
+# exterior_algebra constructor:
+#  args should be underlying CoeffRing and number of indets
+#  optional keyword arg: indet_prefix (must be roman-alphabetic"
+
+
+# Returns 2 components: ExtAlg, list of the gens/variables in order (e1,..,en)
+# [[depends on what pbw_algebra returns!]]
+@doc Markdown.doc"""
+    exterior_algebra_naive(CoeffRing, NumVars)
+    exterior_algebra_naive(CoeffRing, NumVars; var_prefix="x")
+    exterior_algebra_naive(CoeffRing, ListOfVarNames)
+
+First two forms return an exterior algebra with given `CoeffRing` and number of variables.
+By default the variables are called `e1, e2, ...`; the optional keyword parameter can
+specify an alphabetic prefix to use instead of `e`.
+
+The third form returns an exterior algebra with given `CoeffRing`, and variables named
+as specified in `ListOfVarNames` (which must be non-empty).
+
+**Note:** in the first two forms the parameter `NumVars` must be positive; be aware that
+large values will create an object occupying a lot of memory (probably cubic in `NumVars`).
+
+# Examples
+```jldoctest
+julia> ExtAlg,(e1,e2) = exterior_algebra_naive(QQ, 2);
+
+julia> e2*e1
+-e1*e2
+
+julia> is_zero((e1+e2)^2)
+true
+
+julia> ExtAlg,(x1,x2) = exterior_algebra_naive(QQ, 2; var_prefix="x");
+
+julia> x2*x1
+-x1*x2
+
+julia> ExtAlg,(x,y) = exterior_algebra_naive(QQ, ["x","y"]);
+
+julia> y*x
+-x*y
+```
+"""
+function exterior_algebra_naive(K::Ring, NumVars::Int64;  var_prefix="e")
```

Comment:
```suggestion
function exterior_algebra_naive(K::Ring, NumVars::Int;  var_prefix="e")
```
I think it's good to be more generic here.

Reply:
Thanks! Good point!  I plan to accept this suggestion after my next push.

Reply:
Oops!  I thought this could be done automatically, but evidently not after pushing.  Sorry about that.  Anyway, I have adopted your suggestion -- thanks!